audb/core/utils.py

```diff
@@ -35,6 +35,18 @@ def lookup_backend(
     return _lookup(name, version)[1]
 
 
+def move_file(
```

Comment:
Would it make sense to have this as a function in `audeer`? It's not obvious to use `os.replace()` to avoid problems on Windows.

Reply:
We can also first add it there together with `install_packages()` from https://github.com/audeering/audobject/pull/35#pullrequestreview-888439678 before we continue here.

Reply:
Yes, that makes sense